lib/cn/kvset.c

```diff
@@ -2713,64 +2782,32 @@ kvset_iter_set_start(struct kv_iterator *handle, int start, int pt_start)
 }
 
 void
-kvset_madvise_kblks(struct kvset *ks, int advice, bool blooms, bool leaves)
+kvset_madvise_hblk(struct kvset *ks, const int advice, const bool leaves)
 {
-    int i, j;
-
     assert(advice == MADV_WILLNEED || advice == MADV_DONTNEED);
 
```

Comment:
nit: I think it will increase readability if we use local vars to store kh_hblk_desc and kh_ptree_desc. Same applies for kvset_madvise_kblks(). Your call...

Reply:
Think I am just going to keep it since the kblock variant is the same. Can be changed in another PR. IMO our code would be a lot more readable if we stopped doing the completely unnecessary `<pfx>_actual_name` garbage in structs. We shouldn't restrict the quality of the code base to editors of the 80s. In VSCode, all I have to do is right click a struct member and `Find All References` to see all the members. I have told the emacs users among us that they can get this functionality too... Instead that want a symbol that is globally unique for no reason. Very annoying.